site/src/pages/index.js

```diff
@@ -53,8 +53,18 @@ const App = () => {
               <BadgeGrid  />
             </section>
             <div className="banner-logos">
-              <h4> Take pride and embed your Layer5 badges in your GitHub profile</h4>
-              <img src={githubBadges} alt="Recognition Program Banner"/>
+              <h3> Take pride and embed your Layer5 badges in your GitHub profile</h3>
```

Comment:
```suggestion
              <h3> Wear your badge pride proudly. Share and embed your Layer5 badges in your GitHub profile</h3>
```

Reply:
I have stolen this phrase from the Layer5 cloud profile badge card actually :P